src/style.css

```diff
@@ -0,0 +1,115 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 32px 0 16px;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__photo {
+  margin-bottom: 40px;
+  height: 134px;
+}
+
+.photo {
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  padding: 0 16px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  display: flex;
+  margin-bottom: 16px;
+  width: 166px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+  width: 166px;
+  font-size: 10px;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.card__stars--5 {
+  background-image: url(./images/star.svg);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+  width: 166px;
+  height: 18px;
+}
+
+.card__priceText {
+  color: #616070;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+.card__priceN {
+  color: #060b35;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.button {
+  width: 166px;
```

Comment:
you don't need to set width property every time, just set horizontal padding in .card selector